samples/Catalog/Catalog.android.js

```diff
@@ -103,6 +103,22 @@ var examples = [
       component.props.navigation.navigate("PdfView");
     }
   },
+  {
+    name: "PDF View Component Listeners",
+    description:
+      "Show how to use the listeners exposed by PSPDFKitView component.",
+    action: component => {
+      component.props.navigation.navigate("PdfViewListenersScreen");
+    }
+  },
+  {
+    name: "PDF View Instant JSON",
```

Comment:
In https://github.com/PSPDFKit/react-native/pull/101/files#diff-c242db1b34aac23f8a3a7c48ccc0085cR128 we named it 'Programmatic Annotations'. How about something like 'Instant JSON Programmatic Annotations'?